client/src/pages/CreateUserProfile.js

```diff
@@ -0,0 +1,219 @@
+import { Flex, WhiteSpace } from "antd-mobile";
+import { Dropdown, Menu } from "antd";
+import React from "react";
+import styled, { css } from "styled-components";
+import TextInput from "../components/Input/TextInput";
+import Checkbox from "../components/Input/Checkbox";
+import Label from "../components/Input/Label";
+import UnderlineDescription from "../components/Input/UnderlineDescription";
+import CustomH1 from "../components/Typography/Title/CustomH1";
+import Title from "../components/Typography/Title";
+import SubmitButton from "../components/Button/SubmitButton";
+import { theme, mq } from "../constants/theme";
+
+import PersonalDataImage from "../assets/create-profile-images/personal_data.svg";
+
+require('typeface-dm-sans');
+require('typeface-work-sans');
+
+const Container = styled.div`
+    display: flex;
+    .form-container {
+        width: 100%;
+        @media only screen and (min-width: ${mq.tablet.narrow.minWidth}) {
+            width: 50%;
+        }
+        @media only screen and (min-width: ${mq.desktop.medium.minWidth}) {
+            width: 60%;
+        }
+    }
+    .image-container {
+        background-color: ${theme.colors.selago};
+        @media only screen and ${mq.phone.wide.max} {
+            display: none;
+        }
+        @media only screen and (min-width: ${mq.tablet.narrow.minWidth}) {
+            width: 50%;
+        }
+        @media only screen and (min-width: ${mq.desktop.medium.minWidth}) {
+            width: 40%;
+        }
+        img {
+            margin-top: 16rem;
+        }
+    }
+    .form-container, .image-container {
+        padding: 1rem;
+    }
+`;
+
+const ProfileFormGroup = styled.form`
```

Comment:
The background color of the form section looks different than on Figma, I think the color it should be Selago.